src/main.ts

```diff
@@ -345,41 +358,73 @@ export default class ReadwisePlugin extends Plugin {
     window.clearInterval(this.scheduleInterval);
     this.scheduleInterval = null;
     if (!milliseconds) {
-      // we got manual option
+      // user set frequency to manual
       return;
     }
-    this.scheduleInterval = window.setInterval(() => this.requestArchive(null, null, true), milliseconds);
+    this.scheduleInterval = window.setInterval(() => this.refreshBookExport(undefined, true), milliseconds);
     this.registerInterval(this.scheduleInterval);
   }
 
-  refreshBookExport(bookIds?: Array<string>) {
-    bookIds = bookIds || this.settings.booksToRefresh;
-    if (!bookIds.length || !this.settings.refreshBooks) {
+  /** Initiates a sync, or queues refresh for later if queue fails.
+   * ALL syncing goes through this function. */
+  async refreshBookExport(

```

Comment:
heh... The decision to make this the only entry point is good codewise, but it makes the name of this function a lil bit outdated... "Refreshing book export" is meaningful in our backend export flow and it is very narrow: it marks some book to be for sure included in the next export. Clearly this function does this only at the end and can return earlier :) So what would be the better name... `triggerSync`? Too close to `startSync`... Maybe `initiateReadwiseSync` ?

Reply:
What do you think of this name/change? fcdfe0e

Thinking is that `syncBookHighlights` -> `startSync` ... though maybe there's even more room for improvement now that I'm writing it out.. Would be nice if the whole flow, like
`syncBookHighlights` -> `queueExport` -> `getExportStatus` -> `downloadExportArchive`

Reply:
:arrow_up: I went ahead and made this change for the sake of async comms - easy to undo/amend if you're not into it.

Reply:
Niiiiice 👍🏻